build-system/tasks/extension-helpers.js

```diff
@@ -755,18 +755,18 @@ async function getBentoBuildFilename(dir, name, mode, options) {
   if (await fs.pathExists(`${dir}/${filename}`)) {
     return filename;
   }
-  const generatedSource = await generateBentoEntryPointSource(name, toExport);
+  const generatedSource = generateBentoEntryPointSource(name, toExport);
   const generatedFilename = `build/${filename}`;
-  await fs.outputFile(`${dir}/${generatedFilename}`, generatedSource);
```

Comment:
It could be because `outputFile` doesn't return the promise here?
https://github.com/jprichardson/node-fs-extra/blob/master/lib/output-file/index.js#L23

Reply:
They do wrap outputFile in `universalify` to support promises: https://github.com/RyanZim/universalify#universalifyfromcallbackfn

Reply:
Missed that, thanks. Anyhoo it'd be worth seeing if switching to `outputFileSync` prevents this error.

Reply:
sure